jincheng/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo-LIst</title>
+  </head>
+  <body>
+    <div class="container">
+      <div id="root">
+        <input type="text" placeholder="Add Tasks" />
+        <button id="push">Add</button>
```

Comment:
Check this html tag for display "Add" button
```suggestion
        <button id="push">Add</button>
```